source/operations/install-deploy-manage/expand-minio-deployment.rst

```diff
@@ -17,6 +17,9 @@ Expansion does not provide Business Continuity/Disaster Recovery (BC/DR)-grade p
 While each pool is an independent set of servers with distinct :ref:`erasure sets <minio-ec-erasure-set>` for availability, the complete loss of one pool results in MinIO stopping I/O for all pools in the deployment.
 Similarly, an erasure set which loses quorum in one pool represents data loss of objects stored in that set, regardless of the number of other erasure sets or pools.
 
+The new server pool does **not** need to be substantially similar in hardware and software configuration to any existing server pool, though this may allow for simplified cluster management and more predictable performance across pools.
```

Comment:
This may lead people to think an expansion pool can have drives that mix type or size.
```suggestion
The new server pool does **not** need to use the same type or size of  hardware and software configuration as any existing server pool, though doing so may allow for simplified cluster management and more predictable performance across pools.
All drives in the new pool **should** be of the same type and size within the new pool.
```